Sprint-3/revise/implement/password-validator.test.js

```diff
@@ -14,3 +14,36 @@ To be valid, a password must:
 
 You must breakdown this problem in order to solve it. Find one test case first and get that working
 */
+
+const passwordValidator = require("./passwordValidator");
+
+
+test('Checks if password has at least one English uppercase letter (A-Z)', () => {
+    const passwords =[];
+    expect(passwordValidator("Bad@11.#YoZ%", passwords)).toBe(true);
```

Comment:
When given a valid password, the function can correctly return true. (This is what your test represents)

When given an invalid password, can the function correctly return false? (This is what I would like you to test)
For examples, a password missing only a digit, a password missing only an alphanumeric character, etc.

If you prepare the tests correctly, you may discover the bug in your code.